brightStars_DM_sprint_Feb2020/BeyondRadius.ipynb

```diff
@@ -0,0 +1,422 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "for tract, dens in zip(tractList, densities):\n",
+    "    # average detection densities around stars in this magnitude bin:\n",
+    "    magDen = np.mean(dens,axis=0)\n",
+    "    magDenStd = np.std(dens,axis=0)\n",
+    "    plt.errorbar(distanceToEdge, magDen, yerr=magDenStd, capsize=2)\n",
+    "    plt.plot(distanceToEdge, np.min(dens,axis=0), c='red', ls='--')\n",
+    "    plt.plot(distanceToEdge, np.max(dens,axis=0), c='red', ls='--')\n",
+    "\n",
+    "    plt.title(\"Tract {}\\n\".format(tract)+\" ({} bright stars)\".format(len(dens)))\n",
+    "    plt.xlabel('Distance to ABOUT 1/6th of mask edge (arcsec)')\n",
```

Comment:
Can this be changed to a more palatable unit?!  This can't be `arcsec` the way I'm reading this...1/6th of a mask edge would be unitless, no?

Reply:
Well, it's the _distance_ from 1/6th of a mask edge, in arcseconds...

Again, this is pretty last-minute-sprint specific; this plot doesn't really make any sense on its own. It's just a serendipitous example of _how_ the plots can be used if we ever decide to go back to smaller bright star masks (for instance because we're coupling it with an extended PSF model!)

I'll explain what's going on with the crazy units in the conclusion blurb.